boards/risc-v/litex/arty_a7/scripts/gnu-elf.ld

```diff
@@ -0,0 +1,132 @@
+/****************************************************************************
+ * boards/risc-v/litex/arty_a7/scripts/gnu-elf.ld
```

Comment:
should we define a common script for executable?

Reply:
It may be tricky, as the beginning of the `.text` and `.data` sections depend on the MMU and address environment configuration. We could try having a more generic script, I'm just not sure on the specific requirements for other architectures.

Reply:
Ok